.github/workflows/samples-tools/pkg/utils/config.go

```diff
@@ -0,0 +1,101 @@
+package utils
+
+import (
+	"encoding/json"
+	"errors"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+)
+
+type Config struct {
+	// Pattern to match filenames or directories.
+	Match []string `json:"match"`
+
+	// Pattern to ignore filenames or directories.
+	Ignore []string `json:"ignore"`
+
+	// Filename to look for the root of a package.
+	Package []string `json:"package"`
+
+	// Packages to always exclude.
+	ExcludePackages []string `json:"exclude-packages"`
+
+	// Actions to run including the given commands.
+	Actions map[string][]struct {
+		Command string   `json:"command"`
+		Args    []string `json:"args"`
+	} `json:"actions"`
```

Comment:
question: As a map, this implies no promise on ordering. Is there a use case for the type of actions that might be configured here? If we don't have specifics, I suggest dropping this for now because we may want more features in a task runner design here.

Reply:
I was using this at some point to configure what the action did, but it turned out to be simpler to configure in the workflow itself, and that way we didn't pay the price of building the tool on every job as well. So I just removed this.